src/nls/sv/strings.js

```diff
@@ -628,4 +628,4 @@ define({
     "DOCS_MORE_LINK"                            : "Läs mer"
 });
 
-/* Last translated for 893c065b715c211526dcd010c0294e12a8683995 */
+/* Last translated for fd3ef2bf9041fc067c47d600df1372725f6f5f04 */
```

Comment:
Also, this does need to be updated if you're just correcting a typo and didn't actually check the entire localization for new strings.